File: c/Ex24.c
#include<stdio.h>
int main (int argc, char const *argv[])
{
char sexo;
printf("Digite o sexo:(M)para masculino e (F)para feminino:");
scanf("%s",&sexo);
while (sexo != 'F' && sexo != 'M'){   
printf("Nao identificado. \nDigite apenas F para feminino ou M para masculino:  ");
scanf("%s",&sexo);
}
printf("Obrigado!");
    return 0;  

}

File: python/Ex24.py
sexo = (input("Digite o sexo: F feminino ou M masculino: "))

while (sexo!=("F") and sexo!=("M")):
    sexo = (input("Invalido! Digite o sexo: F feminino ou M masculino: "))

if (sexo==("M")):
    print ("Você selecionou o sexo Masculino!")

else:
    print ("Você selecionou o sexo Feminino!")